resources/function_help/json/is_selected

```diff
@@ -14,9 +14,9 @@
             "variant": "One 'feature' parameter",
             "variant_description": "If called with a 'feature' parameter only, the function returns true if the specified feature from the current layer is selected.",
             "arguments": [ { "arg": "feature", "description": "The feature which should be checked for selection." } ],
-            "examples": [ { "expression": "is_selected(@atlas_feature)", "returns": "True if the current atlas feature is selected." },
-                          { "expression": "is_selected(get_feature('streets', 'name', 'Main St.')))", "returns": "True if the unique named \"Main St.\" feature on the active \"streets\" layer is selected." },
-                          { "expression": "is_selected(get_feature_by_id('streets', 1))", "returns": "True if the feature with the id 1 on the active \"streets\" layer is selected." }
+            "examples": [ { "expression": "is_selected(@atlas_feature)", "returns": "True if a selected feature on the current layer is the active atlas feature." },
```

Comment:
Of vs. on aside, what is the reasoning behind this change? I think the old wording was very well readable.

Reply:
IMO, the old wording implied that I could run the expression in any layer, because `@atlas_feature` is an (uniquely?) identifiable feature. But if you set atlas coverage on layer A, and use the expression on layer B (eg to trigger a symbology), it returns false, regardless the atlas feature selection. `is_selected(@atlas_feature)` can return true if:
* you are using the coverage layer.
* or (worse and unexpected ?) if you are using layer B and you selected the feature whose $id is equal to @atlas_featureid

I made a report the other day (#40066) but reading and testing it again, i find some inaccuracies but still the result or expectation of this expression is still unclear to me.

Reply:
In this case, what about: "True if the current atlas feature is selected. Note: Only works if this expression is used on the atlas layer."?

I'm also a bit lost, thinking about a use case. "[User] wants to have a completely different appearance on those atlas pages representing a selected feature.". Maybe look at the commit history to find who introduced it and ping for the use case.